### PR TITLE
feat(ui): Add label component

### PR DESCRIPTION
**Pull request by matejminar**
image

### CODE REVIEW

File: src/sentry/static/sentry/app/components/label.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+// refactor this component to accept words like "warning" and "danger"
+// instead of manually passing backgroundColor and textColor.
+type Props = {
+  text: React.ReactNode;
+  textColor: Color;
+  backgroundColor: Color;
+  tooltip?: React.ReactNode;
+};
+
+function Label({text, textColor, backgroundColor, tooltip}: Props) {
```
Comment (matejminar) [inline src/sentry/static/sentry/app/components/label.tsx:19]: Should we call this label? Is it confusing because of form labels?

**Review by priscilawebdev — APPROVED** [review]
Looks great!  🎉

File: src/sentry/static/sentry/app/components/label.tsx (hunk 1 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/label.tsx]: Hm I'm not sure... I thought about "badge" or "chip" but label is still better

**Review by markstory — APPROVED** [review]

**Comment by billyvg** [comment]
Re: naming, does @doralchan 's design systems have anything?

**Comment by matejminar** [comment]
@billyvg Not really, I'll talk to design tomorrow and submit a PR to rename if needed.

**Pull request merged**